frontend/utils/getColorState.ts

```diff
@@ -0,0 +1,26 @@
+export type ColorState = "success" | "warn" | "error" | "info";
+
+export default (oee: number, isPaused: boolean): ColorState => {
+  if(isPaused) {
+    return "info";
```

Comment:
Blue should be reserved for "maintenance required", not a paused state, per NAMUR NE 107.